package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "tuf-js",
-  "version": "0.0.0",
+  "version": "0.0.1-alpha.1",
```

Comment:
Whenever you update the version string you'll also want to do an `npm install` so that the new value is propagated to the `package-lock.json` file.

Reply:
ohhh make sense! updated